### PR TITLE
MAIN - updates buttons

### PR DESCRIPTION
**Pull request by paschalidi**
Happy to present to you 
our 
✨ **n e w**  ✨ **a m a z i n g** ✨ **b u t t o n s** ✨

#### On the details!
* `height` designers told me that would be 3 heights 56 40 and 32. Property `height` now can take three values `lg`,`sm`, `basic`(basic is the default value)
* restricted the already existing property `type` to: `type: PropTypes.oneOf(["primary", "secondary"])`
* instead of hardcoded hex colors this time we use our own `Colors`!
* wrote cool stories about all the different cases

#### g i f s  -  w o w 
anchor-buttons
buttons

### CODE REVIEW

**Comment by paschalidi** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>✅  Get a 🏆 for adding description on this PR. - <i>Celebrate on every step!</i></td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>✅ Get a 👍 for requesting reviewers. - <i>Celebrating every win!</i></td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 7687932fa3f8441905a74d5ef7204b54b3908362
</p>

File: stories/AnchorButton.story.jsx [hunk 1]
```diff
@@ -8,21 +8,56 @@ storiesOf("AnchorButton", module)
       Call to Action
     </AnchorButton>
   ))
+  .add("primary height 56", () => (
```
Comment (Weetbix) [inline stories/AnchorButton.story.jsx]:
```suggestion
  .add("primary height large", () => (
```

File: stories/AnchorButton.story.jsx [hunk 2]
```diff
@@ -8,21 +8,56 @@ storiesOf("AnchorButton", module)
...
+    <AnchorButton height="lg" href="https://www.vivy.com/" target="_blank">
+      Call to Action
+    </AnchorButton>
+  ))
+  .add("primary height 32", () => (
```
Comment (Weetbix) [inline stories/AnchorButton.story.jsx]:
```suggestion
  .add("primary height small", () => (
```

File: stories/AnchorButton.story.jsx [hunk 3]
```diff
@@ -8,21 +8,56 @@ storiesOf("AnchorButton", module)
...
+    </AnchorButton>
+  ))
   .add("secondary", () => (
     <AnchorButton href="https://www.vivy.com/" target="_blank" type="secondary">
       Call to Action
     </AnchorButton>
   ))
-  .add("tertiary", () => (
-    <AnchorButton href="https://www.vivy.com/" target="_blank" type="tertiary">
+  .add("secondary height 56", () => (
```
Comment (Weetbix) [inline stories/AnchorButton.story.jsx]:
```suggestion
 .add("secondary height large", () => (
```

File: stories/AnchorButton.story.jsx [hunk 4]
```diff
@@ -8,21 +8,56 @@ storiesOf("AnchorButton", module)
...
+    <AnchorButton
+      height="lg"
+      href="https://www.vivy.com/"
+      target="_blank"
+      type="secondary"
+    >
+      Call to Action
+    </AnchorButton>
+  ))
+  .add("secondary height 32", () => (
```
Comment (Weetbix) [inline stories/AnchorButton.story.jsx]:
```suggestion
  .add("secondary height small", () => (
```

File: stories/Button.story.jsx [hunk 5]
```diff
@@ -4,9 +4,33 @@ import { Button, Icon } from "../src/index";
 
 storiesOf("Button", module)
   .add("primary", () => <Button>Call to Action</Button>)
+  .add("primary height 56", () => <Button height="lg">Call to Action</Button>)
```
Comment (Weetbix) [inline stories/Button.story.jsx]:
```suggestion
  .add("primary height large", () => <Button height="lg">Call to Action</Button>)
```

File: stories/Button.story.jsx [hunk 6]
```diff
@@ -4,9 +4,33 @@ import { Button, Icon } from "../src/index";
...
+  .add("primary height 32", () => <Button height="sm">Call to Action</Button>)
```
Comment (Weetbix) [inline stories/Button.story.jsx]:
```suggestion
  .add("primary height small", () => <Button height="sm">Call to Action</Button>)
```

File: stories/Button.story.jsx [hunk 7]
```diff
@@ -4,9 +4,33 @@ import { Button, Icon } from "../src/index";
...
+  .add("primary with custom width", () => (
+    <Button style={{ width: 130 }}>Call to Action</Button>
+  ))
+  .add("primary disabled", () => <Button disabled>Call to Action</Button>)
   .add("secondary", () => <Button type="secondary">Call to Action</Button>)
-  .add("tertiary", () => <Button type="tertiary">Call to Action</Button>)
-  .add("quaternary", () => <Button type="quaternary">Call to Action</Button>)
+  .add("secondary height 56", () => (
```
Comment (Weetbix) [inline stories/Button.story.jsx]:
```suggestion
  .add("secondary height large", () => (
```

File: stories/Button.story.jsx [hunk 8]
```diff
@@ -4,9 +4,33 @@ import { Button, Icon } from "../src/index";
...
+    <Button height="lg" type="secondary">
+      Call to Action
+    </Button>
+  ))
+  .add("secondary height 32", () => (
```
Comment (Weetbix) [inline stories/Button.story.jsx]:
```suggestion
  .add("secondary height small", () => (
```

**Review by Weetbix — CHANGES_REQUESTED** [review]
Looks good, 

I would say better to name the stories after the attribute names rather than the implemented value, I made some suggestions

**Comment by paschalidi** [comment]
> I would say better to name the stories after the attribute names rather than the implemented value, I made some suggestions

Absolutely agree. At the beginning I implemented this such as we parse a number in the component but then decided to change it to string, therefore I forgot to change the story names.. 
Good catch! 

Thanks for the review!

**Review by Weetbix — APPROVED** [review]

**Pull request merged**